ecosystem_tests/ecosystem_tests_cli/commands/blueprint_linting.py

```diff
@@ -0,0 +1,132 @@
+########
+# Copyright (c) 2014-2022 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tempfile
+import subprocess
+from git import Repo
+from github import Github
+from ..logger import logger
+from datetime import datetime
+from ...ecosystem_tests_cli import ecosystem_tests
+from ecosystem_cicd_tools.new_cicd.github import with_github_client
+
+FILE_TYPE = ".yaml"
+BLUEPRINT_START = "tosca_definitions_version:"
+
+
+@ecosystem_tests.command(
+        name='blueprint-linting',
+        short_help='validate blueprints in a repo using cfy-lint autofix.')
+@ecosystem_tests.options.github_token
+@ecosystem_tests.options.repo
+@ecosystem_tests.options.org
+@ecosystem_tests.options.pull_request_title
+def blueprint_linting(github_token=None,
+                      repo=None,
+                      org=None,
+                      pull_request_title=None):
+
+    _blueprint_linting(github_token=github_token,
+                       repository_name=repo,
+                       organization_name=org,
+                       pull_request_title=pull_request_title)
+
+
+@with_github_client
+def _blueprint_linting(github_token,
+                       repository_name,
+                       organization_name,
+                       pull_request_title,
+                       *_,
```

Comment:
Since we do use this, we should not do `__['repository']`

```suggestion
                       repository,
                       *_,
```